handlers/fulfilment-date-calculator/src/main/scala/com/gu/supporter/fulfilment/VoucherBooklet.scala

```diff
@@ -12,14 +13,17 @@ import scala.collection.immutable.ListMap
 object VoucherBookletFulfilmentDates {
 
   lazy val VoucherHolidayStopNoticePeriodDays = 1
+  lazy val FulfilmentCutoffDay = DayOfWeek.WEDNESDAY
+  lazy val WeekStartDay = DayOfWeek.MONDAY
 
   def apply(today: LocalDate): ListMap[String, FulfilmentDates] =
     ListMap( // to preserve insertion order, so the file is easier to read
       List(MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY).map(targetDayOfWeek =>
         targetDayOfWeek.getDisplayName(FULL, ENGLISH) -> FulfilmentDates(
-          today,
-          holidayStopFirstAvailableDate(today),
-          holidayStopProcessorTargetDate(targetDayOfWeek, today)
+          today = today,
+          holidayStopFirstAvailableDate = holidayStopFirstAvailableDate(today),
+          holidayStopProcessorTargetDate = holidayStopProcessorTargetDate(targetDayOfWeek, today),
+          newSubscriptionEarliestStartDate = newSubscriptionEarliestStartDate(targetDayOfWeek, today)
```

Comment:
these needn't be named params as the function name is descriptive
```suggestion
          today,
          holidayStopFirstAvailableDate(today),
          holidayStopProcessorTargetDate(targetDayOfWeek, today),
          newSubscriptionEarliestStartDate(targetDayOfWeek, today)
```